packages/ui/src/components/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -53,7 +53,7 @@
         <div
           ref="process"
           class="va-slider__track va-slider__track--selected"
```

Comment:
![2020-09-28_22-08-47](https://user-images.githubusercontent.com/5394573/94475508-d8184e00-01d7-11eb-84cd-558294296afc.gif)

Reply:
will fix

Reply:
@igorprodev1 you can resolve convo if the issue is fixed. No need to wait for me etc.